charts/stable/zalando-postgres-cluster/values.yaml

```diff
@@ -77,7 +77,11 @@ dumpBackup:
       memory: "10Mi"
       cpu: "5m"
 
-# Dissable Deployment and Service
-controllerType: ""
-service:
+# Disable Deployment
+controller:
   enabled: false
+
+# Disable Service
+service:
+  main:
+    enabled: false
```

Comment:
Do you not want to put these in as hardcoded in `common.yaml`?

Reply:
yes, I can do that in common.yaml

Reply:
moved